src/main/java/backend/Season.java

```diff
@@ -1,8 +1,9 @@
 package backend;
 
 import com.google.gson.Gson;
+import com.google.gson.JsonSyntaxException;
 
```

Comment:
Also check out this: http://stackoverflow.com/questions/147454/why-is-using-a-wild-card-with-a-java-import-statement-bad